clientv3/client.go

```diff
@@ -215,6 +216,12 @@ func (c *Client) dialSetupOpts(endpoint string, dopts ...grpc.DialOption) (opts
 	if c.cfg.DialTimeout > 0 {
 		opts = []grpc.DialOption{grpc.WithTimeout(c.cfg.DialTimeout)}
 	}
+	if c.cfg.DialKeepAlive > 0 {
+		opts = append(opts, grpc.WithKeepaliveParams(keepalive.ClientParameters{
+			Time:    c.cfg.DialKeepAlive * time.Second,
```

Comment:
It defaults to 20 seconds. [Doc link](https://godoc.org/google.golang.org/grpc/keepalive#ClientParameters)

Perhaps I should remove `Timeout` altogether, since 20 seconds should be enough for a response.